samza-test/src/test/scala/org/apache/samza/storage/kv/TestKeyValueStores.scala

```diff
@@ -388,32 +412,31 @@ class TestKeyValueStores(typeOfStore: String, storeConfig: String) {
   def testParallelReadWriteSameKey(): Unit = {
```

Comment:
The test tries to test if `RocksDBKeyValueStore` is threadsafe by reading/writing a key from two different threads. 

RocksDB JNI class inherently is thread-safe. Here's the relevant documentation: 

`
  A RocksDB is a persistent ordered map from keys to values.  It is safe for
  concurrent access from multiple threads without any external synchronization
` 

Samza's `RocksDBKeyValueStore` implementation appears to a wrapper over `RocksDB` JNI class and doesn't have any additional state. I think this doesn't test doesn't add much value and can be deleted.

Reply:
Fair. I deleted the test as agreed.